nextflow.config

```
@@ -6,12 +6,17 @@
 ----------------------------------------------------------------------------------------
 */
 
+nextflow.enable.strict = true
+
 // Global default params, used in configs
 params {
 
     // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
+    taxid                      = null
+    viral_pathogen_genome      = "https://raw.githubusercontent.com/nf-core/test-datasets/createtaxdb/data/fasta/haemophilus_influenzae.fna.gz"
```

I would use something like:
```suggestion
    pipelines_testdata_base_path = "https://raw.githubusercontent.com/nf-core/test-datasets/createtaxdb"
    viral_pathogen_genome      = params.pipelines_testdata_base_path + "/data/fasta/haemophilus_influenzae.fna.gz"

```

cf https://github.com/nf-core/tools/pull/2931

---

Thanks! Done